From e84060d21bc23b002acdd60a6c0e71e2b7cc5c1e Mon Sep 17 00:00:00 2001
From: Simon McVittie <smcv@debian.org>
Date: Sat, 18 Jan 2025 22:06:55 +0000
Subject: [PATCH] Fix regression when running 'meson test'

This needs to be either ('quoted', 'words') or qw(space separated words):
mixing the two syntaxes isn't going to work.

Fixes: becd916b "meson: Refactor a conditional to minize the contents of the branches"
Closes: #1093436
---
 lib/Debian/Debhelper/Buildsystem/meson.pm | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/Debian/Debhelper/Buildsystem/meson.pm b/lib/Debian/Debhelper/Buildsystem/meson.pm
index 5ef3012f..6c038ce3 100644
--- a/lib/Debian/Debhelper/Buildsystem/meson.pm
+++ b/lib/Debian/Debhelper/Buildsystem/meson.pm
@@ -141,7 +141,7 @@ sub test {
 			if ($this->get_parallel() > 0) {
 				$options{update_env}{MESON_TESTTHREADS} = $this->get_parallel();
 			}
-			my @meson_cmd = qw(
+			my @meson_cmd = (
 				'meson',
 				'test',
 			);
-- 
2.47.1

